packages/components/notification/src/NotificationItem/NotificationItem.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useCallback } from 'react';
+import { cx } from 'emotion';
+import {
+  CheckCircleIcon,
+  ErrorCircleIcon,
+  WarningIcon,
+  CloseIcon,
+} from '@contentful/f36-icons';
+
+import { Button } from '@contentful/f36-button';
+import { TextLink } from '@contentful/f36-text-link';
+import { CommonProps, Flex, Box } from '@contentful/f36-core';
+import { Heading, Paragraph } from '@contentful/f36-typography';
+
+import { NotificationVariant, NotificationCta } from '../types';
+import { getStyles } from './NotificationItem.styles';
+
+export interface NotificationItemProps extends CommonProps {
+  variant?: NotificationVariant;
+  withCloseButton?: boolean;
+  onClose?: Function;
+  testId?: string;
+  children: React.ReactNode;
+  title?: string;
+  cta?: Partial<NotificationCta>;
+}
+
+const _NotificationItem = (props: NotificationItemProps, ref) => {
+  const {
+    className,
+    children,
+    cta,
+    withCloseButton = true,
+    variant = 'positive',
+    onClose,
+    testId = 'cf-ui-notification',
+    title,
+    ...otherProps
+  } = props;
+
+  const styles = getStyles({ variant });
+
+  const renderIcon = useCallback(() => {
+    const size = title ? 'medium' : 'small';
+    const iconVariants = {
+      positive: <CheckCircleIcon variant={variant} size={size} />,
+      warning: <WarningIcon variant={variant} size={size} />,
+      negative: <ErrorCircleIcon variant={variant} size={size} />,
+    };
+
+    return <Box className={cx(styles.icon)}>{iconVariants[variant]}</Box>;
+  }, [variant, styles.icon, title]);
+
+  const renderContent = () => (
+    <>
+      {title && (
+        <Heading as="h2" className={cx(styles.title)}>
```

Comment:
Do we need `cx` if only one style class is used?

Reply:
I think we need set marginBottom to "none" for both, Heading and Paragraph components here

Reply:
I had them updated on the style to use the correct spacing since it also needed the `font-size`, `line-height` and `color` properties, but we can move from there to here, if that would be better

Reply:
I just see the difference in a screenshot that shouldn't be there.

Reply:
Oh this may be because the new design has bigger spacing compared than the previous one